nautobot/extras/models/groups.py

```diff
@@ -17,16 +18,140 @@
 from nautobot.core.forms.fields import DynamicModelChoiceField
 from nautobot.core.forms.widgets import StaticSelect2
 from nautobot.core.models import BaseManager, BaseModel
-from nautobot.core.models.generics import OrganizationalModel
+from nautobot.core.models.generics import OrganizationalModel, PrimaryModel
 from nautobot.core.utils.config import get_settings_or_config
 from nautobot.core.utils.lookup import get_filterset_for_model, get_form_for_model
 from nautobot.extras.choices import DynamicGroupOperatorChoices
 from nautobot.extras.querysets import DynamicGroupMembershipQuerySet, DynamicGroupQuerySet
-from nautobot.extras.utils import extras_features
+from nautobot.extras.utils import extras_features, StaticGroupModelsQuery
 
 logger = logging.getLogger(__name__)
 
 
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class StaticGroup(PrimaryModel):
+    """A statically defined (as opposed to dynamically calculated) group of objects sharing a common content-type."""
+
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, unique=True)
+    description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    content_type = models.ForeignKey(
+        to=ContentType,
+        on_delete=models.CASCADE,
+        related_name="static_groups",
+        limit_choices_to=StaticGroupModelsQuery(),
+        help_text="The type of object contained in this Static Group.",
+    )
+
+    clone_fields = ["content_type"]
+    is_static_group_associable_model = False
+
+    class Meta:
+        ordering = ["content_type", "name"]
```

Comment:
Wondering if we should just order by name alone - thoughts?

Reply:
I think so. Content type isn't a uniqueness constraint, it's not shown in the string representation of the group and it probably won't be the first column in the list view?